### DISCUSSION TITLE
Is there a way to generated a smaller readme?

### DISCUSSION THREAD

**Discussion by SamPetherbridge** [upptime/upptime#151] [Q&A]
I have run into an issue where by Summary CI is failing to generate the readme file with the below error.

> This API returns blobs up to 1 MB in size. The requested blob is too large to fetch via the API, but you can use the Git Data API to request blobs up to 100 MB in size.: {"resource":"Blob","field":"data","code":"too_large"}

Realise I am pushing the limits of what is possible with this tool by trying to run more than 750 endpoints in a single project. Given I do not use the readme and only refer to the site is there any option to slim down the readme file so that it fits under the API limit?

**Comment by AnandChowdhary** [comment]
This is super interesting to me, I didn't expect anyone would be using @upptime for 700+ endpoints! 😄

Looks like the GitHub API doesn't allow us to send such large bodies to update the README. Our options are:

- Don't use the GitHub API for updating the README, instead use `git push` like in Graphs CI
- Have a "small README mode" where we only have a tiny summary rather than the full table
- Have an option to disable the README summary altogether
- Any other ideas?

Happy to hear your thoughts and add the required features!
